phpstan-baseline.neon

```diff
@@ -15,6 +15,11 @@ parameters:
 			count: 1
 			path: src/Calculator/Luhn.php
 
+		-
+			message: "#^Unreachable statement \\- code above always terminates\\.$#"
```

Comment:
Why ignoring the error instead of fixing it (removing the unreachable code I'm guessing)?

Reply:
See the comment [above](https://github.com/FakerPHP/Faker/pull/772#discussion_r1331584944)

> This is considered an unreachable statement because per the DocBlock above, a definition should be
> 
>     * a `callable`
> 
>     * an `object`
> 
>     * a `string`

It appears unreachable only based on the `@param callable|object|string $definition` DocBlock, but not based on the actual method argument `$definition` type declaration (it has none).